graders/python/python_autograder/pl_execute.py

```diff
@@ -205,13 +205,13 @@ def execute_code(
                 and student_code[key].__dict__["__name__"] == "matplotlib.pyplot"
             ):
                 plot_value = student_code[key]
+
         if not plot_value:
             import matplotlib as mpl
+            import matplotlib.pyplot as plt
```

Comment:
I don't believe reordering this to come before `mpl.use("Agg")` should make a difference; it didn't in my local testing.